css/block.css

```diff
@@ -242,6 +242,53 @@ wb-expression[type=rect]:before{ background-image: url(../images/icon/rect.svg);
 wb-expression[type=string]:before{ background-image: url(../images/icon/string.svg); }
 wb-expression[type=geolocation]:before{ background-image: url(../images/icon/geolocation.svg); }
 wb-expression[type=size]:before{ background-image: url(../images/icon/size.svg);}
+
+/* block background colour by category*/
+wb-expression.control, wb-step.control, wb-context.control{ background-color: hsl(0, 93%, 56%); border-color: hsl(0, 93%, 40%); }
```

Comment:
I think it is OK like this. I like using class names like that too, but "control" and "text" are too generic, better to make the selector more specific.